sphinx/io.py

```diff
@@ -136,12 +123,12 @@ class SphinxI18nReader(SphinxBaseReader):
     Because the translated texts are partial and they don't have correct line numbers.
     """
 
-    transforms = [ApplySourceWorkaround, ExtraTranslatableNodes, CitationReferences,
-                  DefaultSubstitutions, MoveModuleTargets, HandleCodeBlocks,
-                  AutoNumbering, SortIds, RemoveTranslatableInline,
-                  FilterSystemMessages, RefOnlyBulletListTransform,
-                  UnreferencedFootnotesDetector, SphinxSmartQuotes, ManpageLink,
-                  SubstitutionDefinitionsRemover]
+    def get_transforms(self):
+        # type: () -> List[Type[Transform]]
+        unused = [PreserveTranslatableMessages, Locale, RemoveTranslatableInline,
+                  AutoIndexUpgrader, FigureAligner, SphinxDomains, DoctreeReadEvent,
+                  UIDTransform]
```

Comment:
i18n reader still has list of transforms. It would be better if we can remove this also. Any idea?

Reply:
+1 for this implementation.
i18n builder requires different setup and processing from other builders in various code. Because of this, I think it's difficult to focus on transforms and find a good implementation.